src/components/ebay-fullscreen-dialog-deprecated/README.md

```diff
@@ -1,23 +1,23 @@
 <h1 style='display: flex; justify-content: space-between; align-items: center;'>
     <span>
-        ebay-fullscreen-dialog
+        ebay-fullscreen-dialog-deprecated
     </span>
     <span style='font-weight: normal; font-size: medium; margin-bottom: -15px;'>
         DS v2.0.0
     </span>
 </h1>
 
 ```marko
-<ebay-fullscreen-dialog open a11y-close-text="Close Dialog">
+<ebay-fullscreen-dialog-deprecated open a11y-close-text="Close Dialog">
     <@header>Hello World</@header>
     Body content
-</ebay-fullscreen-dialog>
+</ebay-fullscreen-dialog-deprecated>
 ```
 
-** Deprecated ** (Use lightbox-dialog instead)
+** Deprecated will be removed next major ** (Use lightbox-dialog instead)
```

Comment:
To render `Deprecated` in bold the spaces need to be removed between `**` character. Like `**Deprecated ... major**`